config/releases.exs

```diff
@@ -1,16 +1,23 @@
 import Config
 
-producer_config =
-  "SQS_PRODUCER"
+config =
```

Comment:
I think we should just get in the habit of using `config` for this variable so when we ultimately do copy+paste between projects we don't forget to rename these variable. That has caught us a couple times already